elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java

```diff
@@ -12,19 +12,20 @@
 @Component
 public class ElasticSearchAggregatorScheduledTask
 {
-    // TODO: how can we stop it from running after all records are persisted?
     @Scheduled(fixedDelay = 100)
     public void runWithDelay() throws IOException
     {
         List<MinerRecord> records = ElasticSearchAggregator.getInstance().retrieveAllRecords();
         for( MinerRecord esRecord : records )
         {
-            ElasticSearchAggregator.getInstance().
-                    moveRecordFromElasticSearchToMongo( esRecord, MongoUtils.getCollection() );
+            if( !esRecord.equals( new MinerRecord()) )
```

Comment:
There's a bug - a record in  ElasticSearch with all fields == null.
Could not delete this record with ElasticSearch API.